src/OrchardCore.Modules/Orchard.OpenId/README.md

```diff
@@ -103,3 +103,25 @@ OpenID Connect apps require the following configuration.
       "AllowHybridFlow": false
 }
 ```
+
+## Configuring Certificates
+
+### Windows / IIS
+
+Several tools are availble for generating a signing certificate on Windows and/or IIS, for example:
+
++ IIS Server Manager _(offers not much control)_
+    1. Server Certificates
+    2. Create Self-Signed Certificate 
++ PowerShell _(offers full control)_
+    1. `New-SelfSignedCertificate` https://technet.microsoft.com/en-us/itpro/powershell/windows/pkiclient/new-selfsignedcertificate
```

Comment:
Generation tested OK on Win10 and WinSrv2016, will try and use it too. This is undoubtedly going to be copied & pasted without hesitation, so should we reduce the 5 years, perhaps to 825 days? As per the (equally obscure-) new SSL/TLS validity guidelines.